client/src/i18n/en/tree.json

```diff
@@ -127,6 +127,7 @@
     "PAYROLL" : "Payroll",
     "PAYROLL_RUB_CONFIGURATION" : "Payroll Rubric Configuration",
     "PAYROLL_RUB_MANAGEMENT" : "Payroll Rubric Management",
+    "PAYROLL_SETTINGS" : "Payroll Setting",
```

Comment:
I suggest making this "Payroll Settings"  (plural).  It reads better in English.

Reply:
Thx